katalog/velero/velero-base/deployment.yaml

```diff
@@ -49,7 +53,7 @@ spec:
           containerPort: 8085
         resources:
           limits:
-            memory: 256Mi
+            memory: 1Gi
```

Comment:
😥

Reply:
Ye... https://github.com/vmware-tanzu/velero/issues/2840

Reply:
shouldn't the request be raised a little also?